server/app/app.py

```diff
@@ -29,4 +25,13 @@ def create_app(config_name):
 
     return app
 
+def register_extensions(app):
+    """Register flask extensions"""
+    db.init_app(app)
+    bootstrap.init_app(app)
+    jwt.init_app(app)
+    migrate.init_app(app, db)
+
+    return None
```

Comment:
`return None` is technically unnecessary. 

Depends on the code guidelines we would like to adopt.